### ISSUE TITLE
`message` is ignored if method throws `HTTPError`

### ISSUE THREAD

**Issue by yanok** [purescript-hyper/purescript-hypertrout#7]
If some method decides to throw an `HTTPError` the provided `message` is simply ignored.

**Comment by owickstrom** [comment]
Do you mean this `message`: https://github.com/owickstrom/purescript-hypertrout/blob/master/src/Hyper/Trout/Router.purs#L352 ? Could you provide a minimal example to reproduce the problem?

**Comment by yanok** [comment]
No, I mean when method hander returns `Left RoutingError`, the `message` part the error is ignored. Here:  https://github.com/owickstrom/purescript-hypertrout/blob/fd59265f081795fd010abca43bdc2990e186b478/src/Hyper/Trout/Router.purs#L246
